project/AppDependencies.scala

```diff
@@ -2,20 +2,46 @@ import play.sbt.PlayImport.ws
 import sbt._
 
 object AppDependencies {
+
   val compile = Seq(
     ws,
-    "uk.gov.hmrc" %% "bootstrap-play-25"    % "5.3.0",
-    "uk.gov.hmrc" %% "simple-reactivemongo" % "7.30.0-play-25"
+    "uk.gov.hmrc" %% "bootstrap-play-26"    % "1.13.0",
+    "uk.gov.hmrc" %% "simple-reactivemongo" % "7.30.0-play-26"
   )
 
   val test = Seq(
-    "org.scalatestplus.play"  %% "scalatestplus-play"       % "2.0.1"           % "test,it",
+    "org.scalatestplus.play"  %% "scalatestplus-play"       % "3.1.1"           % "test,it",
     "org.scalatest"           %% "scalatest"                % "3.0.7"           % "test,it",
     "com.github.tomakehurst"  %  "wiremock"                 % "1.58"            % "test,it",
     "org.mockito"             %  "mockito-core"             % "2.27.0"          % "test,it",
```

Comment:
these aren't strictly necessary as gg-test pulls all of the relevant dependencies with it